### ISSUE TITLE
Bad error message for `]add Foo, Bar`

### ISSUE THREAD

**Issue by pfitzseb** [JuliaLang/Pkg.jl#589]
```
(v0.7) pkg> add Images, TestImages, ImageInTerminal
  Cloning git-repo `Images,`
ERROR: failed to clone from Images,, error: GitError(Code:ERROR, Class:Net, unsupported URL protocol)
```

Could also just allow that syntax and it shouldn't drastically complicate the implementation since `,` are (hopefully) not allowed in package names.

**Comment by fredrikekre** [comment]
The problem originates from guessing https://github.com/JuliaLang/Pkg.jl/blob/fb761ea6784436b9dfd16878767ef29cf5c73ef7/src/REPLMode.jl#L174 and we should probably catch it there, long before trying to clone from the faulty input?

**Comment by fredrikekre** [comment]
https://discourse.julialang.org/t/pkg-add-several-packages-syntax-and-error-message/13603

**Comment by StefanKarpinski** [comment]
I think we should consider just allowing this syntax since it's pretty common to want to copy-and-paste  the names on an import/using line and want to add them all. Of course, another approach would be just to interactively prompt about adding whenever someone does `using` for something that isn't yet added to the active environment.

**Comment by pdeffebach** [comment]
I just ran into this today setting up a new computer and it was kind of frustrating. It's something that would make first time users have an easier startup experience.

**Comment by 00vareladavid** [comment]
The "commas" issue was solved with #1187 which should be in Julia 1.3

**Comment by fredrikekre** [comment]
fixed by #1187
